### PR TITLE
Add migration to insert reglementaire bijlage type in triplestore

### PR DESCRIPTION
**Pull request by elpoelma**
This PR adds the `http://data.lblod.info/vocabularies/gelinktnotuleren/ReglementaireBijlage` class to the triplestore.

### CODE REVIEW

**Review by claire-lovisa — APPROVED** [review]
I don't know the context, but it executes and the migration looks good so I'm approving, assuming you know better about why it was needed ! :wink:

**Comment by abeforgit** [comment]
We defined a new class in the gn namespace, this is to add it to CV so when the uri is clicked it dereferences
not needed for anything, but it felt like something we "should do" for all our types but have just been ignoring?

**Comment by abeforgit** [comment]
I can't seriously review this, I don't have time to set up this app

File: config/migrations/2022/20221004110503-Reglementaire-Bijlage-Type.ttl [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@prefix dossier: <https://data.vlaanderen.be/ns/dossier#> .
+@prefix gn: <http://data.lblod.info/vocabularies/gelinktnotuleren/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+
+gn:ReglementaireBijlage a owl:Class ;
```
Comment (nvdk) [inline config/migrations/2022/20221004110503-Reglementaire-Bijlage-Type.ttl:6]: also add rdfs:Class

**Comment by cecemel** [comment]
@elpoelma @nvdk please feel free to merge and release this if ready.  (dunno what the context is)
Release commit looks like https://github.com/lblod/app-centrale-vindplaats/commit/f42d5a17126e77b9e5c4318d13163e56cf12f07d
tag, and merge in development

Deploy on bfg and charlie
thanks

**Comment by nvdk** [comment]
@felix some questions before we proceed:
- do entities need a mu:uuid ?
- is it ok to havel (multiple) language tagged labels?

**Comment by cecemel** [comment]
> @felix some questions before we proceed:
> 
>     * do entities need a mu:uuid ?
It's handy to have them
> 
>     * is it ok to havel (multiple) language tagged labels?
should work I think, check I'd  say

**Comment by nvdk** [comment]
can confirm it works, I've updated the PR with some extra metadata and a few extra classes

File: config/migrations/2022/20221004110503-Reglementaire-Bijlage-Type.ttl (hunk 1 above)
Comment (elpoelma) [reply config/migrations/2022/20221004110503-Reglementaire-Bijlage-Type.ttl]: Isn't owl:Class a subclass of rdfs:Class?

**Pull request merged**